proto/osmosis/poolmanager/v1beta1/query.proto

```diff
@@ -30,8 +30,8 @@ service Query {
   rpc EstimateSinglePoolSwapExactAmountIn(
       EstimateSinglePoolSwapExactAmountInRequest)
       returns (EstimateSwapExactAmountInResponse) {
-    option (google.api.http).get =
-        "/osmosis/poolmanager/v1beta1/{pool_id}/estimate/single_pool_swap_exact_amount_in";
+    option (google.api.http).get = "/osmosis/poolmanager/v1beta1/{pool_id}/"
+                                   "estimate/single_pool_swap_exact_amount_in";
```

Comment:
Is this confirmed? Would be really odd if the linter did something that wasn't proto compatible...

Reply:
I'm also not convinced this is a problem. I searched the repo and there are multiple places where this is happening.

If this is indeed an issue, we should investigate fixing the linter IMO as it is almost guaranteed it will get missed in one of the PRs

Reply:
https://github.com/osmosis-labs/osmosis/issues/4780

Reply:
yeah, not sure. Just going off what Matt said. @mattverse have you confirmed that stuff breaks here? what breaks?

Reply:
From what I have manually tested, I have came to a conclusion that these multiple liners are not correctly supported when they are getting converted to rest endpoints. 

On the second look, it might have been just me having effect by other testing variants. Worthwhile re-testing, cannot assure that its 100% not supported